### PR TITLE
Add no systems alert

### PR DESCRIPTION
**Pull request by Hyperkid123**
jira: https://issues.redhat.com/browse/RHCLOUD-13714

screenshot-ci foo redhat com_1337-2021 04 19-11_02_35

### CODE REVIEW

File: src/Components/NoSytemsAlert/index.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  Grid,
+  GridItem,
+  Text,
+  TextContent,
+  Title,
+} from '@patternfly/react-core';
+import { CopyIcon } from '@patternfly/react-icons';
+
+function NoSystemsAlert({ handleClose }) {
+  return (
```
Comment (karelhala) [inline src/Components/NoSytemsAlert/index.js]: Extra `return` statement.

File: src/Routes/Dashboard/index.js [hunk 2]
```diff
@@ -97,6 +99,11 @@ const SamplePage = () => {
         />
       </PageHeader>
       <Main>
+        <Fragment>
+          {systemsCount === 0 && isGuideOpen && (
```
Comment (karelhala) [inline src/Routes/Dashboard/index.js]: I would gate this behind `systemsLoaded` check as well.

File: src/Components/NoSytemsAlert/index.js (hunk 1 above)
Comment (Hyperkid123) [reply src/Components/NoSytemsAlert/index.js]: Its a function :wink:

Comment (karelhala) [reply src/Components/NoSytemsAlert/index.js]: Aaah, why not making it a const? :smile:

Comment (Hyperkid123) [reply src/Components/NoSytemsAlert/index.js]: :smiling_imp: to confuse the hell out of you all :D Do you want me to make it a const?

Comment (karelhala) [reply src/Components/NoSytemsAlert/index.js]: much appreciated!

**Review by karelhala — APPROVED** [review]

**Pull request merged**